plugins/generator-1.19.4/forge-1.19.4/utils/mcelements.ftl

```diff
@@ -33,5 +33,5 @@
 </#function>
 
 <#function toBlockPos x y z>
-    <#return "new BlockPos(" + opt.removeParentheses(x) + "," + opt.removeParentheses(y) + "," + opt.removeParentheses(z) +")">
+	<#return "BlockPos.containing(" + opt.removeParentheses(x) + "," + opt.removeParentheses(y) + "," + opt.removeParentheses(z) +")">
```

Comment:
@AlsoSomeoneElse, is this change 1:1 to old behaviour? It seems to me, but it is important change so just want a second opinion :)

Reply:
It is. Ideally we could check for int markers on all three inputs and use the constructor if they're int

Reply:
Was thinking about this too. @Goldorion could you look into this solution?

Reply:
How can I check for ints inside FTL?

Reply:
int inputs start with `/*@int*/`. I guess you could check x y and z if they start with this value, but @AlsoSomeoneElse may have better suggestion

Reply:
Yeah, check for `/*@int*/` at the beginning of x y and z